tools/d2g/d2gtest/schemas/test_suites.yml

```diff
@@ -48,8 +48,6 @@ properties:
           required:
             - name
             - description
-            - dockerWorkerTaskPayload
-            - genericWorkerTaskPayload
```

Comment:
I think it would have been better to use `oneOf` json schema construct here, so that you could guarantee that either a pair of payloads are included, _or_ a pair of task definitions.